packages/zk-wizard/src/zk-wallet-account.ts

```diff
@@ -786,7 +786,18 @@ export class ZkWalletAccount {
         .eth()
         .add(fee)
         .toString(16),
-      onComplete: () => this.saveOutflow(note),
+      onComplete: async () => {
+        await this.db.transaction(async db => {
+          db.create('PendingDeposit', {
```

Comment:
don't we need to await `db.create()` promise here?

Reply:
No, it's inside a transaction so it returns synchronously (it just queues the operation). You can optionally await it (like inside `this.saveOutflow`) but it's not necessary.